### ISSUE TITLE
Document SDK Wagmi integration

### ISSUE THREAD

**Issue by alexandratran** [MetaMask/metamask-docs#1006]
Document how to use the Wagmi third-party integration with the SDK.

**Comment by colinlowenberg** [comment]
I believe this is the most critical item in the distribution of the SDK. Wagmi is the most used connector library with 736 forks and 5k stars on GitHub. Right now the Wagmi integration with the SDK is limited to an add-on and not part of the main Wagmi connector library. If we document the integration with Wagmi, developers will start to use it and test it. This will result in the Wagmi team accepting our PR, and push them to add the SDK as part of the Wagmi library by default.

**Comment by ezgi-cengiz** [comment]
@colinlowenberg This integration is not live yet, that's why we don't have any documentation. When we go live with Wagmi we will share documents related to our Wagmi integration.

**Comment by Montoya** [comment]
This needs to be done before Feb 23 (ETH Denver) 
https://wagmi.sh/react/api/connectors/metaMask
